cirq-core/cirq/transformers/randomized_measurements.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2024 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from collections.abc import Sequence
+from typing import Any, Literal
+
+import cirq
+import numpy as np
+from cirq.transformers import transformer_api
+
+
+@transformer_api.transformer
+class RandomizedMeasurements:
+    """A transformer that appends a moment of random rotations to map qubits to
```

Comment:
I didn't notice that you also add the measurement operation.
```suggestion
    """A transformer that appends necessery operations to measure in random pauli bases."""
```